packages/flutter/lib/src/material/pickers/calendar_date_range_picker.dart

```diff
@@ -38,14 +35,14 @@ const double _maxCalendarWidthPortrait = 480.0;
 class CalendarDateRangePicker extends StatefulWidget {
   /// Creates a scrollable calendar grid for picking date ranges.
   CalendarDateRangePicker({
-    Key key,
-    DateTime initialStartDate,
-    DateTime initialEndDate,
-    @required DateTime firstDate,
-    @required DateTime lastDate,
-    DateTime currentDate,
-    @required this.onStartDateChanged,
-    @required this.onEndDateChanged,
+    Key? key,
+    DateTime? initialStartDate,
+    DateTime? initialEndDate,
+    required DateTime firstDate,
+    required DateTime lastDate,
+    DateTime? currentDate,
+    required this.onStartDateChanged,
+    required this.onEndDateChanged,
   }) : initialStartDate = initialStartDate != null ? utils.dateOnly(initialStartDate) : null,
        initialEndDate = initialEndDate != null ? utils.dateOnly(initialEndDate) : null,
        assert(firstDate != null),
```

Comment:
Do we still need this assert (and the one below it)? These will never be null, right?